src/app/pages/about/about.component.html

```diff
@@ -184,6 +184,36 @@ <h1 class="mb-4" id="meet-the-team-title">Meet the Team!</h1>
     </div>
     <hr/>
 
+    <!-- Anthony's Row-->
+    <div class="row d-flex">
+
+      <!-- Anthony's Image -->
+      <div class="col-md-2 order-md-2 image-container">
+        <img alt="Picture of Anthony Zhou" src="{{ Anthony.regularLink }}" class="img-fluid regular-picture">
+        <img alt="Silly picture of Anthony Zhou" src="{{ Anthony.sillyLink }}" class="img-fluid silly-picture">
```

Comment:
```suggestion
        <img alt="Picture of Anthony Wang" src="{{ Anthony.regularLink }}" class="img-fluid regular-picture">
        <img alt="Silly picture of Anthony Wang" src="{{ Anthony.sillyLink }}" class="img-fluid silly-picture">
```

Reply:
What do you mean? He's CLEARLY Anthony Zhou!